tap_hubspot/hubspot.py

```diff
@@ -31,19 +31,24 @@ def __init__(
         self.tap_stream_id = tap_stream_id
         self.event_state = event_state
 
+    def streams(
+        self, properties: List, start_date: datetime, end_date: datetime,
+    ):
         self.refresh_access_token()
         if self.tap_stream_id == "companies":
             yield from self.get_companies(properties)
         elif self.tap_stream_id == "contacts":
-            yield from self.get_contacts(properties)
+            yield from self.get_contacts(
+                properties, start_date=start_date, end_date=end_date
+            )
         elif self.tap_stream_id == "engagements":
             yield from self.get_engagements()
         elif self.tap_stream_id == "deal_pipelines":
             yield from self.get_deal_pipelines()
         elif self.tap_stream_id == "deals":
             yield from self.get_deals(properties)
         elif self.tap_stream_id == "email_events":
-            yield from self.get_email_events()
+            yield from self.get_email_events(start_date=start_date, end_date=end_date)
```

Comment:
Why did we change it in the first place? And why go back now? 😄

Reply:
Because I am complicated. Hahaha